### DISCUSSION TITLE
Proposed Feature: Including Docker deployment for local project development

### DISCUSSION THREAD

**Discussion by ariyonaty** [freeCodeCamp/Developer_Quiz_Site#168] [Ideas]
Given the industry's rising trend towards a microservices architecture, along with containerization and cloud computing, I believe the option for contributors to test their changes through Docker would be beneficial.

Proposed changes would include:
- Adding a Dockerfile to the repo
- Updating relevant documentation (i.e.: CONTRIBUTING.md) with instructions on how to use said Dockerfile 

If approved, I'd like to volunteer to incorporate this feature.

**Comment by jdwilkin4** [comment]
Hi @ariyonaty ! 

I am not opposed to setting up Docker for local development. 
Go ahead and create an issue and we will assign it to you. 👍
